---
title: Prediction of Active Site and Distal Residues in E. Coli DNA Polymerase III
  Alpha Polymerase Activity
authors:
- Ramya Parasuram
- Timothy A. Coulther
- Judith M. Hollander
- Elise Keston-Smith
- Mary Jo Ondrechen
- Penny J. Beuning
date: '2018-01-01'
publishDate: '2025-09-05T20:10:26.485158Z'
publication_types:
- article-journal
publication: '*Biochemistry*'
doi: 10.1021/acs.biochem.7b01004
abstract: The process of DNA replication is carried out with high efficiency and accuracy
  by DNA polymerases. The replicative polymerase in E. coli is DNA Pol III, which
  is a complex of 10 different subunits that coordinates simultaneous replication
  on the leading and lagging strands. The 1160-residue Pol III alpha subunit is responsible
  for the polymerase activity and copies DNA accurately, making one error per 105
  nucleotide incorporations. The goal of this research is to determine the residues
  that contribute to the activity of the polymerase subunit. Homology modeling and
  the computational methods of THEMATICS and POOL were used to predict functionally
  important amino acid residues through their computed chemical properties. Site-directed
  mutagenesis and biochemical assays were used to validate these predictions. Primer
  extension, steady-state single-nucleotide incorporation kinetics, and thermal denaturation
  assays were performed to understand the contribution of these residues to the function
  of the polymerase. This work shows that the top 15 residues predicted by POOL, a
  set that includes the three previously known catalytic aspartate residues, seven
  remote residues, plus five previously unexplored first-layer residues, are important
  for function. Six previously unidentified residues, R362, D405, K553, Y686, E688,
  and H760, are each essential to Pol III activity; three additional residues, Y340,
  R390, and K758, play important roles in activity.
---
