---
title: Altering the Allosteric Pathway in IGPS Suppresses Millisecond Motions and
  Catalytic Activity
authors:
- George P Lisi
- Kyle W East
- Victor S Batista
- J Patrick Loria
- Peter E Wright
date: '2017-01-01'
publishDate: '2025-09-05T20:10:25.725480Z'
publication_types:
- article-journal
doi: 10.1073/pnas.1700448114
abstract: Imidazole glycerol phosphate synthase (IGPS) is a V-type allosteric enzyme,
  meaning that its catalytic rate is critically dependent on activation by its allosteric
  ligand, N$prime$-[(5$prime$-phosphoribulosyl)formimino]-5-aminoimidazole-4-carboxamide
  ribonucleotide (PRFAR). The alloste-ric mechanism of IGPS is reliant on millisecond
  conformational motions for efficient catalysis. We engineered four mutants of IGPS
  designed to disrupt millisecond motions and allosteric coupling to identify regions
  that are critical to IGPS function. Multiple-quantum Carr--Purcell--Mei-boom--Gill
  (CPMG) relaxation dispersion experiments and NMR chem-ical shift titrations reveal
  diminished enzyme flexibility and a reshaping of the allosteric connectivity in
  each mutant construct, respectively. The functional relevance of the observed motional
  quenching is confirmed by significant reductions in glutaminase kinetic activity
  and allosteric ligand binding affinity. This work presents relevant conclusions
  toward the control of protein allo-stery and design of unique allosteric sites for
  potential enzyme inhibitors with regulatory or therapeutic benefit. allostery textbar
  NMR textbar community networks textbar millisecond motions T he underlying principles
  of allosteric regulation have been a focal point of enzymology and structural biology
  for decades as studies of allostery have evolved from two phenomenological models
  (1, 2) to recognize structural and conformational ensem-bles that define a broad
  range of enzymatic states (3--6). As a result, a great deal of emphasis has been
  placed on understanding dynamic contributions to allostery (7--12) and factors that
  enable small fluctuations in local conformations of enzymes to propagate information
  over large distances. A well-developed understanding of dynamic allostery opens
  up numerous avenues for insight into protein engineering (13, 14), drug design (15),
  and mechanistic bio-chemistry (16--22). To establish universal principles for relevant
  en-zymes, however, a link between dynamics and function must be made. The heterodimeric
  enzyme imidazole glycerol phosphate syn-thase (IGPS) plays a crucial role in amino
  acid and purine bio-synthesis in bacteria and other microorganisms by sequentially
  catalyzing the hydrolysis of glutamine (Gln) in its HisH subunit and the cyclization
  of the substrate and allosteric effector N$prime$-[(5$prime$-phosphoribulosyl)formimino]-5-aminoimidazole-4-carboxamide
  ri-bonucleotide (PRFAR) in its HisF subunit (Scheme 1). IGPS is a V-type allosteric
  enzyme that is inactive unless PRFAR is bound, and communication between the active
  and effector sites is driven by the enhancement of conformational flexibility, namely,
  fluctua-tions taking place on the millisecond timescale (22--24). Based on NMR and
  computational evidence, these motions are believed to propagate from the PRFAR binding
  site to the glutaminase binding site, thereby disrupting a hydrogen bond between
  V51 and P10 of HisH. This H bond prevents formation of the oxyanion hole site necessary
  for glutamine hydrolysis, and its disruption allows IGPS to conformationally sample
  the activated state. Recently, we have advanced these studies and demonstrated that
  activation of IGPS is dependent on the ability of allosteric effectors to stimulate
  domain-wide motions in the HisF subunit and ligands that induce greater motion within
  the central ($αβ$) 8 barrel are more effective activators of glutaminase chemistry
  (22). NMR and computational community network studies proposed an optimal allosteric
  pathway through networks of HisF amino acid residues that link the PRFAR binding
  site and adjacent flexible loop 1 to residues comprising a central hydrophobic cluster
  essential for catalysis, as well as salt bridge residues at the dimer interface
  (25, 26). These previous results suggest important amino acids in each community
  that, upon mu-tation, should disrupt the IGPS allosteric network. The demon-strated
  importance of the networks suggests these mutations will alter the protein dynamics
  and that the resulting changes to the dynamical network will affect the corresponding
  functional prop-erties of the enzyme. The prior community network analysis showed
  that a feature of PRFAR binding is the strengthening of some communities at the
  expense of others, which involved multiple residues switching be-tween communities.
  (25) In particular, there was an increase in communication between the HisF community,
  f3$prime$, which contained secondary-structure elements f$β$1, f$β$2, f$β$3, f$β$4,
  f$α$2, f$α$2, and h$α$1, and community h2$prime$, which encompassed the glutaminase
  active site and hP10 (SI Appendix, Fig. S1). This community connectivity is the
  most correlated route of communication between HisF and HisH. Throughout the manuscript,
  we distinguish HisF and HisH residues and communities with lowercase f and h, respectively.
  Additional community interactions in the PRFAR activated state occurred between
  community f3$prime$ and f4$prime$, which include residues from f$β$1, f$β$2, f$α$1,
  and loop 1. To examine the impact of pertur-bations to these allosteric communities
  and pathways, we engi-neered four IGPS variants (V12A, K19A, V48A, and D98A) that
  introduce an alanine residue at crucial positions within HisF (Fig. 1) but distant
  to the PRFAR binding site. The first residue, V12 in f$β$1, represents a residue
  from the apo community, f3 that switches to f1$prime$ in the allosterically activated
  enzyme. In this PRFAR acti-vated state, the f1$prime$ community forms a weak network
  with the glutaminase community h2$prime$. The second residue, K19, is located in
  loop 1, in the apo community f3, and switches to the f4$prime$ community in the
  PRFAR-bound state. The third residue, V48, is located in
---
